packages/tocco-util/src/notifier/modules/modalComponentsTemplates/YesNoCancelButtons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
```

Comment:
Why are YesNoCancel and OkCancelButtons two components? 
Cant you make it more generic. Maybe even call if ButtonBar that accepts a list of buttons containing label and callback (and primary)?

Reply:
By creating one template for each type it is more comprehensible but prone to bad implementation of corporate design. As long we have only two templates it is easy to handle it. May it make sense to refactor out single buttons (e.g. YesButton, NoButton, CancelButton) and use them in both templates?

ButtonGroup is the better choice than ButtonBar because it is a flat list of buttons and not a hierarchically list.